### PR TITLE
Add support for placeholders in chunk names

### LINKED ISSUE #19: Infer name from filename
This loader is very nice but it's a bit cumbersome to have to specify the name of the chunk every time to not have a bunch of numbered chunks. Would it be possible to make the loader use the file name by default as name? Like if I do `require('react-router-proxy!../Routes/Rooms')` it would be equivalent to `require('react-router-proxy?name=rooms!../Routes/Rooms')`?

### PR DESCRIPTION
**Pull request by Anahkiasen**
Closes #19

### CODE REVIEW

**Comment by seanadkinson** [comment]
Looks good.  Can you add something to the `README` about how to use it?

**Comment by Anahkiasen** [comment]
Done, tell me if it's clear enough

File: index.js [hunk 1]
```diff
@@ -8,6 +8,10 @@ module.exports = function() {};
 module.exports.pitch = function(remainingRequest) {
     this.cacheable && this.cacheable();
     var query = loaderUtils.parseQuery(this.query);
+    var chunkName = loaderUtils.interpolateName(this, 'routes/' + query.name, {
```
Comment (seanadkinson) [inline index.js]: For backwards compatibility, should we check the existence of a `[` in the name?  Using the example from the readme, if an existing user had `?name=user`, their behavior would now change.  But maybe that doesn't matter.  Thoughts?

Comment (Anahkiasen) [inline index.js]: I'm thinking maybe automatically putting the chunks in a subfolder _by default_ is a bad idea, would maybe be better to just leave behavior unchanged but mention how to do that in the README?

Comment (seanadkinson) [inline index.js]: Yeah, that would be better.

**Comment by Anahkiasen** [comment]
@seanadkinson done

**Comment by seanadkinson** [comment]
Good stuff @Anahkiasen.  Let me pull this down and do some quick testing, and as long as its backwards compatible I'll push an update.

**Comment by seanadkinson** [comment]
Published `v0.5.1`.  Thanks!

**Pull request merged**